### DISCUSSION TITLE
More info on Archiving of checks

### DISCUSSION THREAD

**Discussion by suvkund** [community/community#54438] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

Hello, 

Recently I stumbled upon below setting, and after looking at the documentation, information rpovided was not enough for me. 
![image](https://user-images.githubusercontent.com/102734887/236195151-a7b1f9ea-8215-4e31-b37a-5667a0dc8dd9.png)

I am wondering by checks does it mean the status checks that are enabled on PR? Also since we dont have GitHub actions enabled for us right now does this setting even help us at all?

Any info will be great.

Thanks,
SK

**Comment by GeoffreyFClark** [comment]
That setting lets GitHub automatically archive + delete data from checks... like test results, created by GitHub Actions, or CI services. It helps manage checks data in your repo. If you don't use checks, this setting won't affect you much.

**Reply by suvkund** [reply]
Hello, we do not use GitHub actions but we do use status checks on Pull Request's which are connected to Jenkins & SonarQube. Does this setting help with those?
